src/content/blog/flutter-jobs.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Getting a Flutter Job
 author: eseidel
-description: How to find or hire for a "Flutter job"
+description: Advice on how to find or hire for a "Flutter job"
```

Comment:
Tried to make this a little longer so that it spans two lines and makes it look more uniform with the other cards